cmd/earth/main.go

```diff
@@ -298,8 +299,33 @@ func (app *earthApp) parseConfigFile(context *cli.Context) error {
 		cfg.Git = map[string]config.GitConfig{}
 	}
 
+	err = app.processDeprecatedCommandOptions(context, cfg)
+	if err != nil {
+		return err
+	}
+
+	gitConfig, gitCredentials, err := config.CreateGitConfig(cfg)
+	if err != nil {
+		return errors.Wrapf(err, "failed to create git config from %s", app.configPath)
+	}
+
+	// command line option overrides the config which overrides the default value
+	if !context.IsSet("buildkit-image") && cfg.Global.BuildkitImage != "" {
```

Comment:
I would check if the underlying variable is `== ""` rather than `context.IsSet`, to avoid typo of the flag name.

Reply:
I don't think that'll work because the underlying variable is set to the default value.

Reply:
Ah ok :+1: